### ISSUE TITLE
Not able to train the model 

### ISSUE THREAD

**Issue by sriramsaiamuluru1** [RasaHQ/rasa_nlu#406]
rasa NLU version0.8.10

Used backend / pipeline spacy_sklearn

Windows

Exception: Not all required packages are installed. Please install sklearn

But I already have sklearn
Requirement already satisfied: sklearn in c:\python27\lib\site-packages
Requirement already satisfied: scikit-learn in c:\python27\lib\site-packages (fr
om sklearn)

**Comment by tmbo** [comment]
Are you maybe running in a virtual environment? how do you run rasa? what happens if you open a python shell and import sklearn?

**Comment by tmbo** [comment]
closed due to inactivity - let me know if this is still an issue.

**Comment by andikas** [comment]
Hi @tmbo,

I got that problem too.
when i try to train a data, an error tell me that not all required packages are installed.

![image](https://user-images.githubusercontent.com/25953422/27323951-a53c225e-55cd-11e7-989c-2c0d191d03dd.png)

Please help me with this.
I need some suggestion on this.

Thanks, 
cheers,

**Comment by tmbo** [comment]
@andikas have you tried following the suggestion from the error message and installed the missing requirements?

**Comment by PHLF** [comment]
I think he did 😉 : `Requirement already satisfied: scikit-learn...`

@andikas Please take some time to read [this](https://docs.python.org/3.6/using/windows.html). It may help you.

**Comment by tmbo** [comment]
Right, sorry - didn't see the last part of the output :D

Try running a python interactive shell and `import sklearn`.

**Comment by andikas** [comment]
Hi @tmbo and @PHLF ,

Thanks for your quick response guys.
I just finished install the missing requirements (numpy+mkl).
I got this error when I try to import sklearn from python shell.
And I got it work now.
Rasa NLU with spaCy + sklearn as backend.
Really appreciate your help on this.

And now I want to try sklearn+Mitie as backend.
Hopefully all is well when I install sklearn+mitie. 

Thanks,
Cheers and God Bless
